steal.lic

```diff
@@ -114,13 +117,11 @@ class Steal
   def bin_items
     return if @bin_items.empty?
 
-    walk_to 6017
-    move('tap knocker')
-    move('north')
+    walk_to @bin_location
```

Comment:
Does this work in the Crossing? The last time I tried it go2 didn't support going directly to the bin room.